test/t8n/t8n.cpp

```diff
@@ -148,7 +148,7 @@ int main(int argc, const char* argv[])
         j_result["receiptsRoot"] = hex0x(state::mpt_hash(receipts));
         j_result["txRoot"] = hex0x(state::mpt_hash(transactions));
 
-        std::ofstream{output_dir / output_result_file} << j_result;
+        std::ofstream{output_dir / output_result_file} << std::setw(4) << j_result;
```

Comment:
```suggestion
        std::ofstream{output_dir / output_result_file} << std::setw(2) << j_result;
```
Good, but let's use 2-space indentation.

Reply:
ok